src/test/java/core/basesyntax/serviceimpl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.serviceimpl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static final String REPORT_TITLE_ROW = "fruit,quantity";
+    private ReportService reportService;
+
+    @Before

```

Comment:
```suggestion
    @BeforeClass

```
check other similar cases